modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -40,23 +40,23 @@ image::ex2-exchange-assets.png[]
 
 Exchange prevents resource exhaustion attacks by limiting the number of asset versions that can be published.
 
-The limit is 500 assets for master organizations with trial accounts, and 100 000 (one hundred thousand) assets for other master organizations.
+The limit is 500 assets for root organizations with trial accounts, and 100 000 (one hundred thousand) assets for other root organizations.
 
 This asset count does not include deleted assets or assets generated by Exchange such as xref:to-deploy-using-rest-connect.adoc[Mule 3 and Mule 4 connectors generated automatically from APIs].
 
-When a master organization has reached 80% of its limit, Exchange shows a warning.
+When a root organization has reached 80% of its limit, Exchange shows a warning.
 
-When a master organization has reached its limit, Exchange shows an error. Also, using the *Publish new asset* button on the home page or the *Add new version* button on the asset detail page shows an error explaining that the limit has been reached and no more assets can be added.
+When a root organization has reached its limit, Exchange shows an error. Also, using the *Publish new asset* button on the home page or the *Add new version* button on the asset detail page shows an error explaining that the limit has been reached and no more assets can be added.
 
-Check the limit of your master organization with a CURL command like this:
+Check the limit of your root organization with a CURL command like this:
 
 ----
     curl -X GET \
-      https://anypoint.mulesoft.com/exchange/api/v1/organizations/MASTER_ORGANIZATION_ID/limits \
+      https://anypoint.mulesoft.com/exchange/api/v1/organizations/ROOT_ORGANIZATION_ID/limits \
       -H 'Authorization: bearer ANYPOINT_TOKEN'
 ----
 
-Replace `MASTER_ORGANIZATION_ID` with your master organization ID, and replace `ANYPOINT_TOKEN` with the authorization token with permissions for the master organization.
+Replace `ROOT_ORGANIZATION_ID` with your root organization ID, and replace `ANYPOINT_TOKEN` with the authorization token with permissions for the root organization.
```

Comment:
To avoid double "with"s...:
```suggestion
Replace `ROOT_ORGANIZATION_ID` with your root organization ID, and replace `ANYPOINT_TOKEN` with the authorization token that has permissions for the root organization.
```